assets/js/common/Pagination/Pagination.jsx

```diff
@@ -38,6 +38,23 @@ const containerClassNames = 'flex items-center';
 const defaultItemsPerPageOptions = [10, 20, 50, 75, 100];
 const defaultItemsPerPage = 10;
 
+function ItemsOnPageStats({
+  currentItemsPerPage,
+  itemsPresent,
+  itemsTotal,
+  selectedPage,
+}) {
+  const itemsBase = (selectedPage - 1) * currentItemsPerPage;
+  const lowerBound = itemsBase + 1;
+  const upperBound = itemsBase + itemsPresent;
+
+  return (
+    <span className="mr-4 text-gray-600">
+      {lowerBound}–{upperBound} of {itemsTotal}
+    </span>
+  );
+}
+
```

Comment:
Could you move this to another file?

Reply:
Also I think you wanna give this a `text-sm` class in order to make the text a little bit smaller

Reply:
I agree with @dottorblaster here. The text size need to be smaller so that it not disproportional in the table footer section and that it also matches the size as the "**Results per page**" text.